### ISSUE TITLE
change 'canvasScaleLine' position 

### ISSUE THREAD

**Issue by joseahr** [Viglino/ol3-ext#4]
I want to modify the default behaviour of canvasScaleLine to place it in the bottom-right corner.
Looking at the source code there's no property provided for aligning it.
If I'm not wrong the parts of the code that handles the position are:

```
var container = $(this.getMap().getViewport()).parent();
var scx = container.outerWidth() / container.get(0).getBoundingClientRect().width;
var scy = container.outerHeight() / container.get(0).getBoundingClientRect().height;
position.left *= scx;
position.top *= scy;
```
But i'm not able to place it in the right place.
Is there a way to manage this?
Thanks in advance and thank you for your work!!

**Comment by Viglino** [comment]
Helo,
The canvasScaleLine control is drawn at the position of the `.ol-scale-line` div (just like the standard scaleLine control). No need to write code, you can change the position directly in your css:
```
.ol-scale-line {
    left:auto;
    right: 8px;
}
```
You also can also change it programmatically (with a render on them map to redraw):
```
$('.ol-scale-line').css ({ left:"auto", right: "8px"});
map.render();
```

**Comment by joseahr** [comment]
Thank you very much! map.render() does the trick!!

**Comment by JURASINTERGEO** [comment]
Hi Viglino. I know that this subject is closed but I've got the same problem as joseahr.
When I change the position of canvasScaleLine into bottom center of the map  in my css as .ol-scale-line object it doesn't take any effect :(
Then I tryed to do it as You suggested "$('.ol-scale-line').css ({ left:"auto", right: "8px"});
map.render();" but I think I'm not such a good in js to set it up properly.
What should I do to place canvasScaleLine in the center - bottom on the map?
